### PR TITLE
feat: NAV-BAR

### PR DESCRIPTION
**Pull request by duolok**
This PR includes navigation bar component for easier navigation.
pic-window-231124-2014-03

### CODE REVIEW

File: src/app/core/core.module.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    NavBarComponent,
+    HeaderComponent,
+    ContainerComponent,
+  ],
+  imports: [
+    CommonModule,
+  ],
+  exports: [
+    NavBarComponent,
+    HeaderComponent,
```
Comment (vradovic) [inline src/app/core/core.module.ts:19]: I can't seem to find the HeaderComponent, for this the build fails :thinking:

File: src/app/core/core.module.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+@NgModule({
+  declarations: [
+    NavBarComponent,
+    HeaderComponent,
+    ContainerComponent,
+  ],
+  imports: [
+    CommonModule,
```
Comment (vradovic) [inline src/app/core/core.module.ts:15]: Import SharedModule instead

File: src/app/startup/startup.module.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HeroComponent } from './hero/hero.component';
+
+@NgModule({
+  declarations: [
+    HeroComponent
+  ],
+  imports: [
+    CommonModule
```
Comment (vradovic) [inline src/app/startup/startup.module.ts:10]: Also import SharedModule here instead of CommonModule

**Review by vradovic — CHANGES_REQUESTED** [review]
Please fix the missing header component and merge conflicts. Aside from that it looks good.

**Comment by duolok** [comment]
You're correct. I was developing multiple components at once and then I forgot to delete some junky imports. I'm not sure what SharedModule is but it is used on a component that won't be used so I'm marking this as resolved and merging PR.

**Pull request merged**